docs/rules/jsx-no-target-blank.md

```diff
@@ -20,7 +16,8 @@ This rule aims to prevent user generated links from creating security vulnerabil
 
 * allow-referrer: optional boolean. If `true` does not require `noreferrer`. Defaults to `false`.
 * enabled: for enabling the rule. 0=off, 1=warn, 2=error. Defaults to 0.
-* enforce: optional string, 'always' or 'never'
+* enforceDynamicLinks: optional string, 'always' or 'never'
+* trustSpreadAttributes: optional boolean. Defaults to `true`.
```

Comment:
i can't decide if this is fine, or if it'd be better to stick with the normal best practice of making "absence" and "false" be the same (and thus, inverting the name, eg `warnOnSpreadSyntax`)

Reply:
ah, i wasn't aware of that pattern! i'll change to `warnOnSpreadSyntax` to make it consistent